airflow/utils/action_loggers.py

```diff
@@ -0,0 +1,101 @@
+# -*- coding: utf-8 -*-
```

Comment:
If this action_loggers is CLI-specific we may want to call the module `cli_action_loggers`

Reply:
Note that if this action_logger was a class (instead of a module) it could actually live in `utils/cli.py`, but here you're using the module scope as a bit of a namespace so in that context it needs its own module.

Reply:
Let me rename it to cli_action_loggers.py

Agree that action_loggers.py could have implemented as a class. I wanted to make it use like a singleton -- so that registered callback will be used all across -- and I think using module is one of the simplest way to do it.